app/controllers/posts_controller.rb

```diff
@@ -93,6 +93,17 @@ def drafts
     render :index
   end
 
+  def random
+    path =
+      if post = Post.published.sample
```

Comment:
hey @jwworth this kind of approach load all posts in memory converts to an array and then fetches a sample of that right? Maybe you could create a scope in `Post